charts/k8s-monitoring/values.yaml

```diff
@@ -228,6 +228,12 @@ metrics:
     # See https://grafana.com/docs/agent/latest/flow/reference/components/prometheus.relabel/#rule-block
     extraMetricRelabelingRules: ""
 
+    cardinalityImprovements:
```

Comment:
Just a thought but `cardinalityImprovements` also can increase cardinality depending on the values, e.g. by including more metrics. Suggest maybe `cardinalityModifications` or simply `cardinality`?